watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,400 @@ users_with_insificient_disk:
         usage: "LABEL"
     - cht_instance:
         usage: "LABEL"
+
+users_with_insufficient_chrome_version:
+  query: |
+    SELECT
+      chrome.partner_name,
+      user_name,
+      chrome_version,
+      required_chrome_version,
+      replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM
+      public.app_monitoring_users_with_insufficient_chrome AS chrome
+    LEFT JOIN
+      monitoring_urls ON chrome.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - user_name:
+        usage: "LABEL"
+    - chrome_version:
+        usage: "GAUGE"
+    - required_chrome_version:
+        usage: "LABEL"
+    - cht_instance:
+        usage: "LABEL"
+
+feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        source,
+        count(*) AS source_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    GROUP BY
+        feedback.partner_name,
+        source,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - source:
+        usage: "LABEL"
+    - source_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+manual_feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'manual'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+automatic_feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'automatic'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+replication_by_status:
+  query: |
+    SELECT
```

Comment:
this query returned 42k rows some of them with bad dates (`"2001-08-25"` ??) and old dates (`"2020-02-02"`) - should we limit this to maybe "last 90 days" instead?

Reply:
Limited to last 90 days

Reply:
much better!  

though...did the table get truncated?  I'm only seeing ~900 rows without the 90 day limit - seems odd :thinking: 

But, let's resolve and move on!